openedx/core/djangoapps/content/learning_sequences/api/data.py

```diff
@@ -251,6 +257,12 @@ class is a pretty dumb container that doesn't understand anything about how
     #   not be able to access anything inside.
     accessible_sequences = attr.ib(type=Set[UsageKey])
 
+    # Unfortunately, have to do this to avoid
+    # ValueError: No mandatory attributes allowed after an attribute with a default value or factory.
+    course_visibility = attr.ib(
+        type=str, default="private", validator=attr.validators.in_(COURSE_VISIBILITIES)
```

Comment:
@ahmedbilal, @ormsbee, would it be okay to drop the default value and expect it to be set by the code that instantiates this class, so that we don't have to duplicate the field definition in this derived class?

Reply:
Sure. Nothing else uses this yet, so backwards compatibility isn't an immediate concern.